test/unit/toUint8Array.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const assert = require('assert');
+
+const sharp = require('../../lib');
+const fixtures = require('../fixtures');
+
+describe('toUint8Array', () => {
+  it('reusing same sharp object does not reset previously passed parameters to toUint8Array', (done) => {
```

Comment:
Thanks for adding tests to cover the output. Please can you add more to cover both callback and Promise based outputs as well as tests for Uint8Array as input, including Stream-based input.

There should be plenty of examples you can use in https://github.com/lovell/sharp/blob/master/test/unit/io.js

Reply:
Alright, I'll look into that, although this test file is just a copy of the toBuffer.js test file. I don't fully understand the project and testing structure of sharp, so this will probably take me way longer than it would take anyone else with experience of sharp and/or unit tests of a big project, though.

Reply:
Hm, so I probably should just search for every test that uses `toBuffer` and copy it but with `Uint8Array`, right?